cli/commands/logs/index.ts

```diff
@@ -0,0 +1,54 @@
+import { AwilixContainer } from 'awilix';
+import { CommandHandler } from "../..";
+import { assertExists, assertIsISOString, assertIsNonEmptyString, isValidTimeRange } from "../../utils";
+import { GetAgentLogs } from '../../utils/get.agent.logs';
+
+
+export default function provideLogs(
+  container: AwilixContainer,
+  agentId: string,
+  getAgentLogs: GetAgentLogs,
+  args: any,
+): CommandHandler {
+  assertExists(container, 'container')
+  assertExists(args, 'args')
+  assertIsNonEmptyString(agentId, "agentId");
+
+
+  return async function logs(cliArgs: any = {}) {
```

Comment:
we can remove unused cliArgs variable too (args *is* the arguments passed in from cli)

Reply:
Removed this variable